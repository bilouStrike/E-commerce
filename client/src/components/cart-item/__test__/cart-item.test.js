import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import CartItem from '../cart-item.component';

it('expect to render cart item component', () =>{
    expect(toJson(shallow(<CartItem />))).toMatchSnapshot();
});