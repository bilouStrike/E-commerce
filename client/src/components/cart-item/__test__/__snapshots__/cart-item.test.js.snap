// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render cart item component 1`] = `
<div
  className="cart-item"
>
  <img
    alt="item"
  />
  <div
    className="item-details"
  >
    <span
      className="name"
    />
    <span
      className="price"
    >
       x $
    </span>
  </div>
</div>
`;
